Makefile

```diff
@@ -42,7 +42,9 @@ patch-nvml-h: $(PKG_BINDINGS_DIR)/nvml.h
 $(PKG_BINDINGS_DIR)/nvml.h: $(GEN_BINDINGS_DIR)/nvml.h | $(PKG_BINDINGS_DIR)
 	sed -E 's#(typedef\s+struct)\s+(nvml.*_st\*)\s+(nvml.*_t);#\1\n{\n    struct \2 handle;\n} \3;#g' $(<) > $(@)
 
-bindings: $(PKG_BINDINGS_DIR)/nvml.h $(SOURCES) | $(PKG_BINDINGS_DIR)
+bindings: .create-bindings .strip-autogen-comment
```

Comment:
It probably doesn't matter, but will this break if we ever use `make -j` since `.create-bindings .strip-autogen-comment` will run in parallel?

Reply:
It shouldn't since `.strip-autogetn-comments` has an ordering dependency:
```
.strip-autogen-comment: | .create-bindings
```